www/sdk.ts

```diff
@@ -49,14 +64,14 @@ class Sdk {
     /**
      * Reports the update status to the CodePush server.
      */
-    public static reportStatus(status: string, callback?: Callback<void>) {
+    public static reportStatus(status: string, pkg: IPackage, callback?: Callback<void>) {
```

Comment:
The status and pkg arguments should be optional. https://github.com/Microsoft/code-push/pull/101/files#diff-a937c637a47cbd31cbb52c89bef7d197R143 If unspecified, this will report to the server that the app is running a new binary version with no CodePush installed package. See my other comment for more information.